src/pretix/base/forms/questions.py

```diff
@@ -488,26 +488,18 @@ def __init__(self, *args, **kwargs):
         for q in questions:
             # Do we already have an answer? Provide it as the initial value
             answers = [a for a in pos.answerlist if a.question_id == q.id]
-            if answers:
-                initial = answers[0]
-            else:
-                initial = None
+            initial = answers[0] if answers else None
             tz = pytz.timezone(event.settings.timezone)
             help_text = rich_text(q.help_text)
-            label =  mark_safe(q.question)  # django-bootstrap3 calls mark_safe
+            label = escape(q.question)
             required = q.required and not self.all_optional
             if q.type == Question.TYPE_BOOLEAN:
                 if q.required:
-                    # For some reason, django-bootstrap3 does not set the required attribute
-                    # itself.
                     widget = forms.CheckboxInput(attrs={'required': 'required'})
                 else:
                     widget = forms.CheckboxInput()
 
-                if initial:
-                    initialbool = (initial.answer == "True")
-                else:
-                    initialbool = False
+                initialbool = (initial.answer == "True") if initial else False
```

Comment:
i am talking about this line which preserves the logic of "required"

Reply:
yes,but the logic of implementing required is preserved

Reply:
Does "preserved" mean that you don't change anything related to "required" (the logic of "required" will be the same before and after this PR)? 

Then you should update the PR title and description, to not mislead people into thinking that this PR is about to change the "required".

Your phrase 
> 'required' attribute is only set when 'q.required ' is 'True'

is very confusing, because looking into #174 and this PR, I don't see anywhere mentioning "required attribute" or "`q.required`". Please don't bring something out-of-context here.

Reply:
The PR description can just be "Fixes #174".

Reply:
> 

> The PR description can just be "Fixes #174".

sure i will rephrase the PR description.